### PR TITLE
example test

### PR DESCRIPTION
**Pull request by Sammyeri**
im saying hello, during a test..

### CODE REVIEW

**Review by Dylan12734 — COMMENTED** [review]
the changes are fine

**Review by mayaaraquil — COMMENTED** [review]
Good job!

**Review by Sammyeri — COMMENTED** [review]
hello again

File: midtermproject/midtermproject/Program.cs [hunk 1]
```diff
@@ -1,2 +1 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿Console.WriteLine("Hello1");
```
Comment (Dylan12734) [inline midtermproject/midtermproject/Program.cs:1]: I really like this one!

Comment (Sammyeri) [inline midtermproject/midtermproject/Program.cs:1]: hello me!

Comment (mayaaraquil) [inline midtermproject/midtermproject/Program.cs:1]: Please be nice when i'm requesting my changes TT

**Review by Dylan12734 — APPROVED** [review]

**Review by mayaaraquil — APPROVED** [review]

**Pull request merged**